src/__tests__/services/mail.js

```diff
@@ -16,69 +16,68 @@ chai.use(chaiHttp);
 chai.use(sinonChai);
 
 describe('Email services', () => {
-    describe('Should fail to send mail', async () => {
-        beforeEach(async () => {
-            try {
-                sgMail.setApiKey(fakeApiKey);
-            } catch (err) {
-                throw err;
-            }
-        });
+  describe('Should fail to send mail', async () => {
+    beforeEach(async () => {
+      try {
+        sgMail.setApiKey(fakeApiKey);
```

Comment:
why are you editing the indentations?

Reply:
I linted before pushing. I guess eslint did that as they were many eslint errors